.github/workflows/changelog.yml

```diff
@@ -24,7 +24,7 @@ jobs:
       run: |
         echo $FILES | grep -qF 'CHANGELOG.md' || echo $PR_BODY | grep -qiF "$NO_CHANGELOG"
     - name: Reject pull request if no CHANGELOG update
-      if: ${{ always() && steps.changelog_updated.outcome == 'failure' }}
+      if: ${{ always() && steps.changelog_updated.outcome == 'failure' && github.actor != 'dependabot' }}
```

Comment:
I think we should add this at the job level, as we don't want to run any step for dependabot. I think it should be like:
```
jobs:
  check_changelog:
    if: github.actor != 'dependabot'
    runs-on: ubuntu-20.04
    env:
      NO_CHANGELOG: '[x] No CHANGELOG update needed'
```


https://github.blog/changelog/2019-10-01-github-actions-new-workflow-syntax-features

Reply:
addressed in bed2e37